### ISSUE TITLE
20.03 Upgrade - qutebrowser and beets packages are broken

### ISSUE THREAD

**Issue by bunwayenterprises** [NixOS/nixpkgs#85980]
Reasonably new to NixOS so please forgive any stupid questions :)

**Describe the bug**
I upgraded to release 20.03 a couple of days ago and I am currently unable to install qutebrowser or beets. Both give errors relating to broken python libraries relating to the same nix store path of:

`/nix/store/i30lxs79vw0mxwcw8yv3z6v3wb6k29ji-nixos-20.03.1445.95b9c99f6d0`

**To Reproduce**

Either:

- Specify qutebrowser and beets in configuration.nix and run `nixos-rebuild switch`

or

```
nix-env -iA nixos.qutebrowser
nix-env -iA nixos.beets
````

And receive the following:
```
[~] nix-env -iA nixos.qutebrowser
installing 'qutebrowser-1.10.0'
error: Package ‘python3.7-stem-1.7.1’ in /nix/store/i30lxs79vw0mxwcw8yv3z6v3wb6k29ji-nixos-20.03.1445.95b9c99f6d0/nixos/pkgs/development/python-modules/stem/default.nix:25 is marked as broken, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.
```
```
[~] nix-env -iA nixos.beets
installing 'beets-1.4.9'
error: Package ‘python3.7-soco-0.18.1’ in /nix/store/i30lxs79vw0mxwcw8yv3z6v3wb6k29ji-nixos-20.03.1445.95b9c99f6d0/nixos/pkgs/development/python-modules/soco/default.nix:31 is marked as broken, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.
```

**Expected behavior**
Packages install as per 19.09 release

**Additional context**
I'm reasonably new to nixos so unsure what is the reason for them not working out of the box.

Upgraded from **19.09** with **system.stateVersion = "19.09"**

On 19.09 I was running qutebrowser from the unstable channel and once 20.03 was released I switched it to the stable version. I've tried removing it completely, doing a rebuild/switch and then adding it back but it doesn't work.

Currently having to run both beets and qutebrowser from unstable which install fine.

**Notify maintainers**
Not sure which (or if all) of the 60 maintainers should be @'d

**Metadata**
```
[~] nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.33, NixOS, 20.03.1445.95b9c99f6d0 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.4`
 - channels(root): `"nixos-20.03.1445.95b9c99f6d0"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by aristaeus** [comment]
Both packages have a dependency which is broken. They're fixed on the unstable branch though. There's already a [PR backporting the fix for pythonPackages.stem](https://github.com/NixOS/nixpkgs/pull/85642) (the qutebrowser dependency), but I can't see anything for soco.

**Comment by symphorien** [comment]
Not related to the actual issue: you should not change system.stateVersion. See the comment in the default configuration.nix:
https://github.com/NixOS/nixpkgs/blob/d19089e1e7ef569399cd010bffed739ba6ffb1fc/nixos/modules/installer/tools/tools.nix#L174-L180
and the manual section about upgrading: https://nixos.org/nixos/manual/index.html#sec-upgrading

**Comment by bunwayenterprises** [comment]
@aristaeus - Thanks for that, I searched for relevant issues in the PR backlog but this one seemed to have slipped past me. Looks like i'll wait for it to become approved, merged, and backported before switching qutebrowser back from unstable.

@symphorien - My configuration.nix state is still at 19.09 currently.

**Comment by symphorien** [comment]
Ah sorry I misread you sentence "upgraded with system.stateVersion".

**Comment by bunwayenterprises** [comment]
Closing as fixed in the latest 20.03 update.
